client/platform/desktop/backend/native/common.ts

```diff
@@ -88,9 +88,9 @@ async function _findCSVTrackFiles(originalBasePath: string) {
 /**
  * locate json track file in a directory
  * @param path path to a directory
- * @returns absolute path to json file or null
+ * @returns absolute path to json file or empty string
  */
-async function _findJsonTrackFile(basePath: string): Promise<string | null> {
+async function _findJsonTrackFile(basePath: string): Promise<string> {
```

Comment:
`string | null` I think when we were relying two difference pieces of information before.
1. If it auto found nothing you got Null which meant there wasn't one located in the loading folder
2. If the value was `''` emtpy string it meant the user intentionally removed the suggested file and didn't want to load it

This was kept around before because of the stereoscopic loading.  I removed all sense of auto-finding stereoscopic annotations and make the user import directly the files that exist.  The auto finding is now done before actual loading so the user either gets back an empty string or changes the track file to an empty string by clearing it.

I also didn't want to be passing null to the v-text-fields, I feel better keeping them as always strings.

Through double checking this I did notice another mistake I'll be pushing up soon (I accidently set some stereo annotation files to null instead of '')